README Update for Logging Changes (#59)
TomWhitwell/SlowMovie: README.md

                        adjust image contrast (default: 1.0)
  -l, --loop            loop a single video; otherwise play through the files
                        in the videos directory

>   -o {DEBUG,INFO,WARNING,ERROR,CRITICAL}, --loglevel {DEBUG,INFO,WARNING,ERROR,CRITICAL}
>                          minimum importance-level of messages saved to the
>                          logfile (default: INFO)

Reviewer: ```suggestion
  -o {DEBUG,INFO,WARNING,ERROR,CRITICAL}, --loglevel {DEBUG,INFO,WARNING,ERROR,CRITICAL}
                         minimum importance-level of messages displayed and
                         saved to the logfile (default: INFO)
```

Updated for new message re: 8f411a8729b4167d4496599b401717b6728a99d4

> Args that start with '--' (eg. -f) can also be set in a config file
> (slowmovie.conf). Config file syntax allows: key=value, flag=true,

Reviewer 2: ```suggestion
Arguments can also be set in a config file (slowmovie.conf). Config
file syntax allows: key=value, flag=true,
```
I tried using short arguments in the config file, and it works. (I would've aligned the rest of this paragraph, but GitHub doesn't allow commenting on collapsed lines.)
